### PR TITLE
Homework 01 YM

### PR DESCRIPTION
**Pull request by yunyanmo**
Name: Yunyan Mo

I wrote a short biography about myself in R markdown. As someone that is using markdown for the first time, most of the process was relatively easy. Following the Mastering Markdown Git guide provided on the assignment page, I was able to apply the required elements such as italics, bold, header, etc. pretty successfully. The challenges came when I tried to embed an image into the markdown file. At first, I thought I did everything correctly and had already push the homework to Github, but as I check over it before submitting, I found that the image I thought I had inserted could not be found. Thus, I went back and tried reinsert for several times until I finally discover where went wrong -- I was not putting in the correct path to this image on my computer. Then, I look up how to find the location of a file on Google; following what I've read online, I opened Finder, went to Go on the tool bar, and then to the "Go to Finder" function, I dragged the image file to that search bar, and it was able to give me the right location of the image, for which I used to fix this problem in my md file. 

Overall, I think this is a pretty good first assignment; being able to fix an issue successfully is always the most satisfying part.

### CODE REVIEW

**Comment by pyctina** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      ✓      |              |            |
| **Coding strategy**         |       ✓    |              |            |
| **Presentation**            |       ✓    |              |            |
| **Achievement, creativity** |           |       ✓       |            |
| **Reproducibility**         |      ✓     |              |            |

Great job on the assignment! You have included all the requirements. The headers are used correctly, bold texts are included, the movie list is listed properly, the link works, and the picture is embedded nicely. The code is clear and concise; it runs without any errors. I see that you have committed multiple times, which is an excellent habit to have. 
three things that you can improve upon are: 
 * You can aim to go above and beyond to try to include new elements other than the ones taught in this course. 
 * try to add more varieties on the list (including ordered and unordered) 
 * the sentence included inside the [ ] when you embed the picture was not shown on the final file. Don't know if you did that intentionally or not; if not, maybe it's good to check the file after you finish to see if everything looks like what you intended to be. 

Overall, you did a really good job!

**Review by psposa — COMMENTED** [review]
Great job figuring out how to add the photo of yourself. You also did a good job with creating the unordered list and using different types of emphasis. The one thing I would point out is that the [Alt text] you set for the photo doesn't show up when you view the image.

**Comment by hanjimin06** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     *      |              |            |
| **Coding strategy**         |     *      |              |            |
| **Presentation**            |     ***      |              |            |
| **Achievement, creativity** |           |     *       |            |
| **Reproducibility**         |     *    |              |            |

Remarks:

My favorite thing about the code was its presentation. I'm not completely sure if this was due to the fact that myself was not organized, but I feel that the paragraph breaks were helpful and also the content was cohesive too. I checked the code on my computer and it worked, so reproducibility worked fine. Also, I noticed that you included all elements mentioned, which is definitely a plus. 

For improvement:

1. Again, others have pointed this out, but the caption does not show up. I'm not the best person to deal with these technical problems, so sorry I could not help.

2. Although this assignment is not the best assignment to exhibit creativity, I believe you could have added some more contents in addition to some general facts about yourself.

3. In addition to hitting all the marks, you could always try for new elements that you could find online.

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+
+I enjoy reading, drawing, and watching movies. Here are some movies that I plan to watch over the summer:
+* Thelma and Louise
+* Paprika
+* 2001: A Space Odyssey
+* Baryy Lyndon
+* Loving Vincent
+* Memoirs of a Geisha
+
+![This is me in Kyoto, Japan, I look forward to the day when we can travel around the world freely again.](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/70931713_107528367318081_2020253329279418368_o.jpg?_nc_cat=102&_nc_sid=09cbfe&_nc_ohc=yMLF4sFhvukAX-i5OLy&_nc_ht=scontent-ort2-1.xx&oh=61ae425a6ba83ccf92219b7f5eaacbdf&oe=5F1983C0)
```
Comment (hanjimin06) [inline README.md:16]: Not visible when I run code.

**Review by hanjimin06 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     *      |              |            |
| **Coding strategy**         |     *      |              |            |
| **Presentation**            |     ***      |              |            |
| **Achievement, creativity** |           |     *       |            |
| **Reproducibility**         |     *    |              |            |

Remarks:

My favorite thing about the code was its presentation. I'm not completely sure if this was due to the fact that myself was not organized, but I feel that the paragraph breaks were helpful and also the content was cohesive too. I checked the code on my computer and it worked, so reproducibility worked fine. Also, I noticed that you included all elements mentioned, which is definitely a plus. 

For improvement:

1. Again, others have pointed this out, but the caption does not show up. I'm not the best person to deal with these technical problems, so sorry I could not help.

2. Although this assignment is not the best assignment to exhibit creativity, I believe you could have added some more contents in addition to some general facts about yourself.

3. In addition to hitting all the marks, you could always try for new elements that you could find online.

File: README.md (hunk 1 above)
Comment (nearridge) [inline README.md:16]: This isn't quite the right way to embed images. It doesn't appear in your repo and doesn't appear for me either. Not sure why. What you can do is you can upload the image to your RStudio server and commit that change to GitHub and then you can reference it normally by the filename!

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+Given my unique name, it is not hard to deduce where did I come from. I was born and raise in **Guangdong, China**. It is a beautiful southern province. If you want to learn more about my hometown, you can visit [this link.](https://en.wikipedia.org/wiki/Guangdong)
+
+I enjoy reading, drawing, and watching movies. Here are some movies that I plan to watch over the summer:
+* Thelma and Louise
+* Paprika
+* 2001: A Space Odyssey
+* Baryy Lyndon
+* Loving Vincent
+* Memoirs of a Geisha
```
Comment (nearridge) [inline README.md:14]: Nice list.

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+## Biography of Yunyan Mo
+
+My name is Yunyan Mo. I usually ask people to just call me *Yan* because it is easier to prounce than my full name and will allieviate the inconviniences and embarassments one might feel when I was first introduced to him/her.
```
Comment (nearridge) [inline README.md:4]: Thanks for the intro!@

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Reproducibility**         |           |          x    |            |

Remarks:

**Pull request closed without merging**